lib/core/src/persist/migrations.rs

```diff
@@ -65,5 +65,6 @@ pub(crate) fn current_migrations() -> Vec<&'static str> {
         ALTER TABLE send_swaps ADD COLUMN description TEXT;
         ALTER TABLE chain_swaps ADD COLUMN description TEXT;
         ",
+        "ALTER TABLE payment_tx_data ADD COLUMN destination_address TEXT;",
```

Comment:
From what I understand, this is only set for chain swaps, and it's set to the swaps `claim_address`. There is already a field called `claim_address` in the `chain_swaps` table.

Instead of inserting it here (duplicating it) in the table for tx data, why not update the `select_payment_query` to read the `claim_address` directly from `chain_swaps`?

Reply:
That's what currently happens for chain swaps (we don't use the `PaymentTxData::destination_address` but just the `claim_address` column from the swap data. 
The reason for creating this is related only with direct onchain (Liquid) payments where we want to rebuild the destination within the `PaymentDetails`. I'll suggest an alternative solution, let me know if it makes more sense.